ReactSkia/views/common/RSkDrawUtils.cpp

```diff
@@ -344,12 +346,14 @@ void drawBorder(SkCanvas *canvas,
 bool  drawShadow(SkCanvas *canvas,Rect frame,
```

Comment:
Add additional input argument "bool shadowOnTransparentComponent". Which to be used before clipping